client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -120,6 +120,12 @@ public class RssSparkConfig {
   // todo: remove unnecessary configuration
   public static final String RSS_CLIENT_BITMAP_SPLIT_NUM =
       SPARK_RSS_CONFIG_PREFIX + "rss.client.bitmap.splitNum";
+  public static final String RSS_CLIENT_MAPSIDE_MERGE_ENABLE =
+      SPARK_RSS_CONFIG_PREFIX + "rss.client.mapside.merge.enable";
+  public static final boolean RSS_CLIENT_MEGE_ENABLE_DEFAULT_VALUE = false;
+  public static final String RSS_CLIENT_MAPSIDE_MERGE_STAGING_SIZE =
+      SPARK_RSS_CONFIG_PREFIX + "rss.client.mapside.merge.staging.size";
```

Comment:
What's the meaning of `staging`?

Reply:
Naming is difficulty. Maybe `rss.client.mapside.merge.max.spill.threshold` is better? 

Do u have any good ideas?

Reply:
`rss.client.mapside.merge.spill.threshold`  is OK